src/FNAPlatform/OpenGLDevice_GL.cs

```diff
@@ -855,7 +855,8 @@ bool enabled
 			if (type == GLenum.GL_DEBUG_TYPE_ERROR)
 			{
 				FNALoggerEXT.LogError(err);
-				throw new InvalidOperationException(err);
+                //On a des exceptions ici � cause de l'overlay steam. Si on commente le throw, �a marche quand m�me.
```

Comment:
Looks like a temp comment got left in...

Reply:
Oh yes I forgot it. There is a throw here if the Steam Overlay is on, but it can be commented and the game still works.